### PR TITLE
Release notes added

### PR DESCRIPTION
**Pull request by TehreemNisa**

### CODE REVIEW

**Comment by stakater-user** [comment]
@TehreemNisa Image is available for testing. `docker pull docker-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-265-773fbeaa`

File: content/sre/release-notes/release-notes.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Release Notes
+
+Below is a monthly report published alongside new or updated software that details the technical features of the Stakater Agility Platform. For new releases, these notes provide end-users with a brief summary of the features. For updates to existing products, the notes educate end-users on how the new version of the software improves upon the former.
```
Comment (rasheedamir) [inline content/sre/release-notes/release-notes.md]: `Stakater Agility Platform` to `Stakater App Agility Platform`

Comment (TehreemNisa) [reply content/sre/release-notes/release-notes.md]: updated

**Comment by stakater-user** [comment]
@TehreemNisa Yikes! You better fix it before anyone else finds out! [Build](https://github.com/stakater/StakaterCloudDocs/commit/fb619a147a1867674894e1f33a6023a556e0fa1f/checks) has Failed!

**Comment by stakater-user** [comment]
@TehreemNisa Image is available for testing. `docker pull docker-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-265-319feee7`

**Comment by stakater-user** [comment]
@TehreemNisa Image is available for testing. `docker pull docker-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-265-4578fa0c`

**Comment by stakater-user** [comment]
@TehreemNisa Image is available for testing. `docker pull docker-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-265-bc1e22ab`

**Comment by stakater-user** [comment]
@TehreemNisa Image is available for testing. `docker pull docker-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:SNAPSHOT-PR-265-8343afe8`

**Review by usamaahmadkhan — APPROVED** [review]

**Comment by stakater-user** [comment]
@usamaahmadkhan Image is available for testing. `docker pull docker-stakater-nexus.apps.prod.clusters.stakater.io/stakater/StakaterCloudDocs:v0.0.200`

**Pull request merged**